.rubocop_todo.yml

```diff
@@ -416,13 +398,6 @@ Style/NestedModifier:
 Style/NestedParenthesizedCalls:
   Enabled: false
 
-# Offense count: 22
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle, MinBodyLength, SupportedStyles.
-# SupportedStyles: skip_modifier_ifs, always
-Style/Next:
```

Comment:
Again, this is is a regular ruby idiom, meant for us to understand when the loop runs or not. Not sure why making that inconsistent is better. Having this cop allows the person reading the code to not think twice if an 'if' condition in a loop is used to skip an iteration